src/api/procedures/addInstruction.ts

```diff
@@ -63,30 +67,30 @@ export interface Storage {
  */
 type InternalAddAndAffirmInstructionParams = [
   u64,
-  SettlementType,
-  Moment | null,
-  Moment | null,
+  PalletSettlementSettlementType,
+  u64 | null,
```

Comment:
Same here, replacing `Moment` with `u64` directly seems a bit risky to me

Reply:
Moment was an alias for u64 like the ID.

I agree with `Moment` being better, but with the previous behavior and TS "structural" type system I think they could be used interchangeable before.